### DISCUSSION TITLE
Context menu goes transparent when there is video on webpage

### DISCUSSION THREAD

**Discussion by 775405984** [valinet/ExplorerPatcher#625] [General]
https://www.dxy.cn/bbs/newweb/pc/post/45865605

Click the webpage above.

only when taskbar was set to windows 10.
version 22000.376.40.21

**Comment by Anixx** [comment]
What context menu and in what browser?

The link does not open.

**Reply by 775405984** [reply]
https://www.dxy.cn/bbs/newweb/pc/post/45865605
open this.
Microsoft Edge
Play video, right click it, context menu is faded.
Screen grab strangely looks normal.
![Snipaste_2022-01-04_17-26-45](https://user-images.githubusercontent.com/7196377/148037875-f3fc8410-2684-4130-9ecf-3a1304b3d5ca.png)

but look at the photo I took:
![微信图片_20220104172619](https://user-images.githubusercontent.com/7196377/148037979-935422ed-91c6-44d2-ac35-1fa12fb72a91.jpg)
![微信图片_20220104172623](https://user-images.githubusercontent.com/7196377/148037983-e2cdb20b-32b4-4bdf-b4bc-74a663de2200.jpg)

**Reply by 775405984** [reply]
Updated to 22000.376.41.0, the problem still persists.

**Reply by Anixx** [reply]
Try running edge with option  --disable-dwm-composition , does it help (before doing it, kill all running Edge processes using Task Manager)?
Does it happen in light theme?
Does it happen when explorer.exe process is not running?
Does it happen when transparency effects are disabled in personalization in settings app?

The menu looks correct for me.

**Reply by valinet** [reply]
EP only injects itself into explorer.exe. Have you tried running and confirming it doesn’t do the same without EP installed? Another test is to have EP installed, enable the Windows 11 taskbar in EP and test then. Also, do you have the program installed as a shell extension? Only then is the DLL injected in other applications. If you did, be careful with the options you choose, things like this *may* happen, I guess. Using as shell extension is **not recommended**, it is only there for some specific scenario. Read more here: https://github.com/valinet/ExplorerPatcher/wiki/Using-ExplorerPatcher-as-shell-extension

**Reply by 775405984** [reply]
Fixed it：Downgrade to Windows 10.
^0^

**Reply by valinet** [reply]
> Fixed it：Downgrade to Windows 10. ^0^

You probably mean "upgrade" ;)
